README.md

```diff
@@ -693,6 +693,18 @@ It's possible to disable persistent caches using the `noStorage` connection opti
 It is also possible to setup one's own persistent cache, by passing a `storage` object as a connection option. This object should implement 3 methods: `getItem`, `setItem`, and `removeItem` (synchronous)
 
 
+## Auto-refresh caches
+
+The SDK includes a mechnism to maintain the schemas and options caches up-to-date by polling the Campaign server on a regular basis (10 seconds by default). The server returns the list of entities (schemas or options) which have changed since they were cached, and the client removes them from the cache.
```

Comment:
we don't mention "method" cache?

Reply:
some cases in which the server will tell the client to clear all the cache:
1. if the number of entities changed is larger than 10
2. build number has been changed

Reply:
Should be ok now